tools/tests/styles/accordion-menu.css

```diff
@@ -41,4 +46,5 @@ gameface-accordion-menu, gameface-accordion-panel, gameface-accordion-header, ga
     border-right: 6px solid transparent;
     right: 24px;
     top: 24px;
+    pointer-events: none;
```

Comment:
why?

Reply:
I've added this in the **component style.css** in this PR because otherwise clicking on the caret/arrow didn't toggle the menu because the event listener is on the parent and it seemed like an easy change. Adding this here for consistency.
![image](https://user-images.githubusercontent.com/21058220/157895822-79ee832a-be33-49f7-bec2-0fbe7f3a27cc.png)